packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -1374,6 +1376,66 @@ describe('RpcContractProvider test', () => {
     });
   });
 
+  describe('originateRollup', () => {
```

Comment:
I would consider changing the name to originateTxRollup to remove possible confusion with the incoming SCORU

Reply:
Good catch, I changed the name from originateRollup to originateTxRollup during the implementation and forgot to correct the tests.